pkgs/desktops/gnome/apps/gnome-weather/default.nix

```diff
@@ -56,6 +56,8 @@ stdenv.mkDerivation rec {
 
     chmod +x meson_post_install.py
     patchShebangs meson_post_install.py
+    substituteInPlace meson_post_install.py \
+      --replace gtk-update-icon-cache gtk4-update-icon-cache
```

Comment:
Looks like this will allow us to switch to `wrapGAppsHook4`.

Reply:
Done.